docs/developer/upgrade-guide/2-11-to-3-0.mdx

```diff
@@ -0,0 +1,213 @@
+---
+title: Saleor 3.0
+---
+
+## Preparation
+
+New version introduces breaking changes and many new features. API clients like Apps and storefronts will require changes. 
+
+Before you start the process, create the backup of your data. Also let your Celery worker to process task queue before starting API migration.
+
+Due to scope of the upgrade, downtime will be required.
+
+## Multichannel
+
+Existing data will be assigned to the `"default-channel"` created during the database migration. As long, as shop has the only one channel defined, you are not required to specify the `channel` argument. Thanks to that you'll be able to migrate storefront to 3.0 granually.
+
+After adding more channels, argument will become required. Because of that, we recommend always adding the channel argument, even if you have only one channel:
```

Comment:
```suggestion
After adding more channels, the `channel` argument will become required. Because of that, we recommend always adding the channel argument, even if you have only one channel:
```